# submissions

---

## Alternative Splicing (SAJR)

For submitting samples for alternative splicing analysis you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/QizTdN5XDw7exsB) and edit the "alternativeSplicing" and "samples" sheets according to the "example" provided in the excel file. 

___

## ATAC-Seq  and ChIP-Seq

For submitting samples for ATAC-Seq or ChIP-Seq analysis you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/dBCH7UJjuQ4POWS) and edit the "ATAC_seq" and "samples" sheets according to the "example" provided in the excel file. If you have input samples for normalizing your ChIP-seq samples, please provide the information in the "input" sheet.

---

## circRNA

For submitting samples for circRNA analysis you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/JG3vcXbC38YAaaA) and edit the "circRNA" and "samples" sheets according to the "example" provided in the excel file. 

---

## Intron retention

For submitting samples for intron retention analysis you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/IxrSwl39WKtJ0Zw) and edit the "intronRet" and "samples" sheets according to the "example" provided in the excel file. 


---

## IRfinder

For submitting samples for intron retention analysis using **IRfinder** you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/cwhfo0RNhfGyxVT) and edit the "IRfinder" and "samples" sheets according to the "example" provided in the excel file. 


---

## miRNA

For submitting samples for intron retention analysis using **miRNA** you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/J524uoDJW3c0axG) and edit the "miRNAseq" and "samples" sheets according to the "example" provided in the excel file. 

---

## Ribo Sequencing

For submitting samples for Ribo-Seq analysis you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/peNmwgGYuy43GYA) and edit the "riboseq" and "samples" sheets according to the "example" provided in the excel file. If you have total RNA samples for normalizing your Ribo-Seq samples, please provide the information in the "matching" sheet.

---

## RNAseq

For submitting samples for RNAseq analysis you will need to copy your raw files into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/exhJ4KWfkgGmJrH) and edit the "RNAseq" and "samples" sheets according to the "example" provided in the excel file.

___

## Variant Calling

For submitting samples for variant calling analysis you will need to copy your raw file into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

If you have exome sequencing please provide a bed file with the targeted exons [example](https://datashare.mpcdf.mpg.de/s/XWTiVZ5RQQZmkVK)

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/880qluG3WOY7CJc) and edit the "variantCalling" and "samples" sheets according to the "example" provided in the excel file. If you have a WT for normalization please specify the respecive sample in the "samples" sheet according to the "example".

---

## 16S Microbial Sequencing

For submitting samples for 16S microbial sequencing analysis you will need to copy your raw files into store-age.age.mpg.de/coworking/group_bit_all/automation. 

Make sure you create a folder eg. `John_Doe_sequencing_March2020` and that all your `fastq.gz` files are inside as well as your md5sums file (attention: only one md5sums file per project).

All files will have to be on your project folder (ie. `John_Doe_sequencing_March2020`) do not create further subfolders.

Once all the files have been copied, download the submission template [here](https://datashare.mpcdf.mpg.de/s/VVyDUCa1oOIaBxl) and edit the "16S" and "samples" sheets according to the "example" provided in the excel file.

**Note reads are annotated using Silva database**

---

## Files from tape storage

For re-submitting samples already on tape please write "TAPE" in the field "Folder".  
You will get a file with a list of used files with the uploaded results to check if the correct files where used.  

___

## GEO/SRA

If you want to analyse GEO/SRA data you can do this without downloading the respective files by giving in the SRA run number instead of the file name. If you are analysing paired end data and only one run number exists please give in the same run number in "Read 2". In the "Folder" and "md5sums" fields you will need to give in "SRA". An example can be found [here](https://youtu.be/KMtk3NCWVnI). 

___

If you have problems downloading your fastq.gz and md5sum files please contact the organization that sent you the files and ask them for support.

If you have problems accessing store-age.age.mpg.de/coworking/group_bit_all/submissions please contact it@age.mpg.de. 

If you have problems filling up the submission template please contact bioinformatics@age.mpg.de.

All files in store-age.age.mpg.de/coworking/group_bit_all/automation will be deleted after 30 days.
